xbdd-web/src/App.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import TagListContainer from "./modules/tag-list/containers/TagListContainer";
+import Report from "./models/Report";
+import "./App.css";
+import tagListData from "./resources/tag-response.json";
+
+const App = () => {
+    const dummyReport = new Report(tagListData);
```

Comment:
We should add TODOs for the stuff we want to remove later?

Reply:
kk